### ISSUE TITLE
Scam domains

### ISSUE THREAD

**Issue by iam-py-test** [uBlockOrigin/uAssets#10124]
### Prerequisites

- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues) to avoid opening a duplicate issue
- [X] I am running the latest version of uBlock Origin

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### URL(s) where the issue occurs

```
youtuba.com
polyhymnia-mar.com
trktrk.club
virpropcnow.xyz
```

### Describe the issue

These domains are scams and seem to be unblocked by uBlock badware.
See https://github.com/DandelionSprout/adfilt/issues/287 for details

### Screenshot(s)

![](https://user-images.githubusercontent.com/84232764/135685234-42cb6776-9928-473f-a4be-bdb584e896ae.png)

![](https://user-images.githubusercontent.com/84232764/135685249-7a09e298-92d5-4299-a724-081778191378.png)

![](https://user-images.githubusercontent.com/84232764/135685262-b4a2020d-76a8-4bb4-b489-677cc17e3d73.png)

### uBlock Origin version

uBlock Origin 1.37.2

### Browser name and version

Firefox 92.0.1 (on host machine) 

### Settings

- Advanced user mode: Enabled
- Dynamic filtering: Hard mode
- All built-in, Ads, malware domains, and multipurpose lists: Enabed
- All privacy lists except AdGuard URL Tracking Protection: Enabled
- All annoyances: Disabled
- NOR, DNK, ISL: Dandelion Sprouts nordiske filtre: Enabled
- Custom filters:
![image](https://user-images.githubusercontent.com/84232764/135761235-34e077f8-2f6f-489d-bedd-51d01e93ebf8.png)

### Notes

All the urls involved in the scam redirects: [redirect_urls_1.txt](https://github.com/uBlockOrigin/uAssets/files/7273786/redirect_urls_1.txt)
This was tested on Microsoft Edge in a VM
Some of these domains are Tor-aware, so using Tor Browser will cause them to appear parked, while a non-Tor IP reveals their true nature
CCing @DandelionSprout

**Comment by iam-py-test** [comment]
@mapx- thanks for the quick response!
